elasticsearch/templates/statefulset.yaml

```diff
@@ -216,3 +216,50 @@ spec:
         {{- if .Values.extraVolumeMounts }}
 {{ tpl .Values.extraVolumeMounts . | indent 10 }}
         {{- end }}
+      {{- if eq .Values.roles.master "true" }}
+      - name: graceful-termination-handler
+        image: "{{ .Values.image }}:{{ .Values.imageTag }}"
```

Comment:
This is not to use additional docker image. If there's any recommendation of light image with `curl`, we can use it.